Source/SIMPLib/Plugin/SIMPLibPluginLoader.cpp

```diff
@@ -151,6 +160,25 @@ void SIMPLibPluginLoader::LoadPluginFilters(FilterManager* filterManager, bool q
       {
         if(!quiet) qDebug() << "Could not set the working directory.";
       }
+      if(!quiet)
+      {
+        qDebug() << "Adding Folder " << thePath;
+      }
+    }
+    if(aPluginDir.cd("lib"))
+    {
+      thePath = aPluginDir.absolutePath();
+      pluginDirs << thePath;
+      aPluginDir.cdUp(); // Move back up a directory level
+      int no_error = chdir(aPluginDir.absolutePath().toLatin1().constData());
+      if(no_error < 0)
+      {
+        if(!quiet) qDebug() << "Could not set the working directory.";
+      }
+      if(!quiet)
```

Comment:
I am not entirely certain, but this looks like it should be an `else if` considering the preceding error check.

Reply:
Adjusted order to make it clearer when error/status messages are printed.